docs/process_joined_data.js

```diff
@@ -55,42 +58,68 @@ const ALWAYS_PRESENT_KEYS = Object.freeze([geoidTag, blockGroupTag, scoreTag]);
  * adds/overwrites the computed attributes of score and color. We avoid doing a
  * full copy because it would unnecessarily copy a lot of data.
  *
- * @param {Promise} dataPromise
+ * @param {Promise<Array<GeoJsonFeature>>} dataPromise
  * @param {number} scalingFactor multiplies the raw score, it can be
  *     adjusted to make sure that the values span the desired range of ~0 to
  *     ~100.
- * @param {Promise<Object>} initialTogglesValuesPromise promise
+ * @param {Promise<ScoreComputationParameters>} scoreComputationParameters
+ *     promise
  * that returns the poverty and damage thresholds and the poverty weight (from
  * which the damage weight is derived).
- * @typedef {Object} GeoJsonFeature See
- *     https://macwright.org/2015/03/23/geojson-second-bite.html#features
  * @return {Promise<{featuresList: Array<GeoJsonFeature>, columnsFound:
- *     Array<string>}>} Resolved scored features, together with all columns
- *     found in features
+ *     Array<EeColumn>}>} Resolved scored features, together with all columns
+ *     found in features (and the additional {@link scoreTag} and
+ *     {@link COLOR_TAG} columns). The first four columns are: {@link geoidTag},
+ *     `districtDescriptionKey` from {@link ScoreParameters},
+ *     {@link scoreTag}, and then `povertyRateKey` from
+ *     {@link ScoreParameters}. If damage is present in the features,
+ *     {@link damageTag} is next, followed by `buildingKey` from
+ *     {@link ScoreParameters}. If damage is absent, `buildingKey` is last.
+ *     Remaining columns are sorted by order encountered in the feature, which
+ *     generally means alphabetically, since EarthEngine always sorts them.
  */
 function processJoinedData(
-    dataPromise, scalingFactor, initialTogglesValuesPromise) {
-  return Promise.all([dataPromise, initialTogglesValuesPromise])
+    dataPromise, scalingFactor, scoreComputationParameters) {
+  return Promise.all([dataPromise, scoreComputationParameters])
```

Comment:
super nit: I think we've been sticking to appending these vars with Promise when they're promises